TeachingRecordSystem/src/TeachingRecordSystem.SupportUi/Pages/Shared/PersonLayout.cshtml

```diff
@@ -0,0 +1,37 @@
+@using TeachingRecordSystem.Core.Dqt.Models;
+@using TeachingRecordSystem.SupportUi.Pages.Common;
+@{
+    Layout = "_Layout";
+    var selectedTab = (PersonSubNavigationTab)ViewBag.SelectedTab;
+    var personId = (Guid)ViewBag.PersonId;
+    var search = (string)ViewBag.Search;
+    var sortBy = (ContactSearchSortByOption)ViewBag.SortBy;
+    var pageNumber = (int)ViewBag.PageNumber;
+}
+
+@section BeforeContent {
+    <govuk-back-link href="@LinkGenerator.Persons(search, sortBy, pageNumber)" />
```

Comment:
Do we want this in the layout - isn't this back link page specific?

Reply:
They all go back to the results screen